src/main/java/org/recordrobotics/charger/commands/auto/AutoMoveClaw.java

```diff
@@ -35,21 +36,12 @@ public AutoMoveClaw(Claw claw, double speed, int status){
 	 */
 	@Override
 
-	public void initialize() {
+	public void execute() {
+		SmartDashboard.putBoolean("switch state", _claw.getSwitchState());
 		if (_status == _grab) {
```

Comment:
`_status` is misleading.. shouldn't it be `_desiredState` or `_command` or something?